online/vite.config.js

```diff
@@ -1,8 +1,7 @@
 import { defineConfig } from "vite";
-import react from "@vitejs/plugin-react";
+import rescript from "@jihchi/vite-plugin-rescript";
 
-// https://vitejs.dev/config/
 export default defineConfig({
-  plugins: [react()],
-  base: "/rescript-json-schema",
+  base: process.env.CI ? "https://dzakh.github.io/rescript-json-schema/" : "/",
```

Comment:
Is the change needed? If yes, maybe it should be something like this?
```suggestion
  base: process.env.CI ? "/rescript-json-schema/" : "/",
```

Reply:
It might still work, but I don’t have any confidence 🤷‍♂️

Reply:
I just confirmed that it works in my repository, so I updated the PR.